nautobot/dcim/tables/template_code.py

```diff
@@ -62,11 +62,39 @@
 """
 
 INTERFACE_IPADDRESSES = """
-{% for ip in record.ip_addresses.all %}
+{% for ip in record.interface.ip_addresses.all %}
```

Comment:
Why did we add `.interface ` here? feel like this is gonna cause an error on `BaseInterfaceTable`

Reply:
thanks - fixed